packages/offix-client/test/createOptimisticResponseTest.ts

```diff
@@ -1,37 +0,0 @@
-import { createOptimisticResponse, MutationHelperOptions, CacheOperation  } from "offix-cache";
-import { expect, should } from "chai";
-import { CREATE_ITEM, GET_ITEMS } from "./mock/mutations";
-
-describe("Optimistic Response tests", () => {
```

Comment:
?? :>

Reply:
because these tests are in the cache package now, not the client - I just forgot to move this file last time.